
 execute mkoffsets via Qemu and build mkbc with the host compiler.

--- w-openjdk-6-1.orig/icedtea6-1.10/patches/arm.patch	2011-02-23 23:32:32.909601440 +0100
+++ w-openjdk-6-1/icedtea6-1.10/patches/arm.patch	2011-08-09 20:05:56.721765260 +0200
@@ -25,7 +25,7 @@
 +
 +offsets_arm.s:	mkoffsets
 +	@echo Generating assembler offsets
-+	./mkoffsets > $@
++	$(QEMU) ./mkoffsets > $@
 +
 +bytecodes_arm.s: bytecodes_arm.def mkbc
 +	@echo Generatine ARM assembler bytecode sequences
@@ -33,12 +33,12 @@
 +
 +mkbc:	$(GAMMADIR)/tools/mkbc.c
 +	@echo Compiling mkbc tool
-+	$(CC_COMPILE) -o $@ $< $(COMPILE_DONE)
++	$(CC_FOR_BUILD) -o $@ $< $(COMPILE_DONE)
 +
 +mkoffsets:	asm_helper.cpp
 +	@echo Compiling offset generator
 +	$(QUIETLY) $(REMOVE_TARGET)
-+	$(CC_COMPILE) -DSTATIC_OFFSETS -o $@ $< $(COMPILE_DONE)
++	$(CC_COMPILE) -static -DSTATIC_OFFSETS -o $@ $< $(COMPILE_DONE)
 +
 +endif
 +
